cmake/SetupRajaOptions.cmake

```diff
@@ -15,6 +15,8 @@ option(RAJA_ENABLE_TBB "Build TBB support" Off)
 option(RAJA_ENABLE_TARGET_OPENMP "Build OpenMP on target device support" Off)
 option(RAJA_ENABLE_SYCL "Build SYCL support" Off)
 
+option(RAJA_ENABLE_EXPTVECTOR "Build experimental vectorization support" Off)
```

Comment:
Please remove 'EXPT' from the variable name so users don't have to change that when we move it out of the 'expt' namespace. Also, I think it should be 'RAJA_ENABLE_VECTORIZATION' so it's more clear.